end2end.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/playwright:v1.44.0-jammy
```

Comment:
Need to make this match the version used in `end2end/package.json` instead of using the latest tag.